assets/src/amp-validation/amp-validated-url-post-edit-screen.js

```diff
@@ -389,13 +392,10 @@ const handleBulkActions = () => {
 
 		const markingAsReviewed = target.classList.contains( 'reviewed' );
 
-		[ ...document.querySelectorAll( 'select.amp-validation-error-status' ) ].forEach( ( select ) => {
-			const row = select.closest( 'tr' );
-			if ( row.querySelector( '.check-column input[type=checkbox]' ).checked ) {
-				row.querySelector( 'input[type=checkbox].amp-validation-error-status-review' ).checked = markingAsReviewed;
-				row.classList.toggle( 'new', ! markingAsReviewed );
-				addBeforeUnloadPrompt();
-			}
+		getURLValidationTableRows( { checkedOnly: true } ).forEach( ( row ) => {
```

Comment:
This is a tangential change. I found myself copying pasting this code (which is from another PR I worked on recently) and instead created the `getURLValidationTableRows` helper function.